lib/components/user/nav-login-button.css

```diff
@@ -25,6 +25,18 @@
   background-color: #080808;
 }
 
+.navBarItem > p {
+  padding: 5px 15px;
+  margin: 0;
+  font-weight: 600;
+  font-size: 16px;
+}
+
+.navBarItem > ul {
+  list-style: none;
+  padding-inline-start: 0;
+}
+
```

Comment:
These `>` are targeting the direct children of `.navBarItem` and neither of these are direct children 

<img width="177" alt="image" src="https://github.com/user-attachments/assets/39d97fb7-0852-4efc-b5c2-198bb5ab6a80">